src/makielayout/types.jl

```diff
@@ -1589,3 +1589,129 @@ end
+        tellwidth = true
+        "Controls if the parent layout can adjust to this element's height"
+        tellheight = true
+        "The horizontal alignment of the scene in its suggested bounding box."
+        halign = :center
+        "The vertical alignment of the scene in its suggested bounding box."
+        valign = :center
+        "The alignment of the scene in its suggested bounding box."
+        alignmode = Inside()
+        "The background color of the axis."
+        backgroundcolor = :white
+        "The maximum radius of the PolarAxis. This acts as the limit of the axis."
+        radius = nothing
+        "The direction of rotation. Can be -1 (clockwise) or 1 (counterclockwise)."
+        direction = 1
+        "The angular offset for (1, 0) in the PolarAxis. This rotates the axis."
+        theta_0 = 0f0
+        "The width of the spine."
+        spinewidth = 2
+        "The color of the spine."
+        spinecolor = :black
+        "Controls whether the spine is visible."
+        spinevisible = true
+        "The linestyle of the spine."
+        spinestyle = nothing
+        "The specifier for the radial (`r`) ticks, similar to `xticks` for a normal Axis."
+        rticks = LinearTicks(4)
+        "The specifier for the minor `r` ticks."
+        rminorticks = IntervalsBetween(2)
+        "The color of the `r` grid."
+        rgridcolor = inherit(scene, (:Axis, :xgridcolor), (:black, 0.5))
+        "The linewidth of the `r` grid."
+        rgridwidth = inherit(scene, (:Axis, :xgridwidth), 1)
+        "The linestyle of the `r` grid."
+        rgridstyle = inherit(scene, (:Axis, :xgridstyle), nothing)
+        "Controls if the `r` grid is visible."
+        rgridvisible = inherit(scene, (:Axis, :xgridvisible), true)
+        "The formatter for the `r` ticks"
+        rtickformat = Makie.automatic
+        "The fontsize of the `r` tick labels."
+        rticklabelsize = inherit(scene, (:Axis, :xticklabelsize), 16)
+        "The font of the `r` tick labels."
+        rticklabelfont = inherit(scene, (:Axis, :xticklabelfont), inherit(scene, :font, Makie.defaultfont()))
+        "The color of the `r` tick labels."
+        rticklabelcolor = inherit(scene, (:Axis, :xticklabelcolor), inherit(scene, :textcolor, :black))
+        "The width of the outline of `r` ticks. Setting this to 0 will remove the outline."
+        rticklabelstrokewidth = 1.0
+        "The color of the outline of `r` ticks. By default this uses the background color."
+        rticklabelstrokecolor = automatic
+        "Controls if the `r` ticks are visible."
+        rticklabelsvisible = inherit(scene, (:Axis, :xticklabelsvisible), true)
+        "The angle in radians along which the `r` ticks are printed."
+        rtickangle = π/8
+        "The specifier for the angular (`theta`) ticks, similar to `yticks` for a normal Axis."
+        thetaticks = ((0:45:315) .* pi/180, ["$(x)°" for x in 0:45:315])
```

Comment:
`MultiplesTicks` doesn't like to increment by 45° so I switched to manual ticks here.